src/routes/index.svelte

```diff
@@ -138,6 +138,7 @@
     const deleteSymbol = symbol => {
         history.delete(symbol)
         history = history
+        highlightedSymbolIndex = null
```

Comment:
What if we did `$: { history.size && resetHighlightedSymbol() }

```
const resetHighlightedSymbol = () => {
   highlightedSymbolIndex = null
}
```

Reply:
Good idea 👍